mm2src/coins/rpc_command/account_balance.rs

```diff
@@ -27,51 +26,71 @@ pub struct AccountBalanceParams {
 }
 
 #[derive(Debug, PartialEq, Serialize)]
-pub struct HDAccountBalanceResponse {
+pub struct HDAccountBalanceResponse<BalanceMap> {
```

Comment:
`BalanceMap` is not necessary a map. Maybe just `Balance` or `BalanceHolder`?

Reply:
How about `BalanceObject`?

Reply:
Fixed here https://github.com/KomodoPlatform/komodo-defi-framework/pull/1979/commits/09c12606641d9bcb6799ae6bb1d7b35e37f74169